build.gradle.kts

```diff
@@ -1,18 +1,19 @@
-plugins {
-    `kotlin-dsl`
-}
-
 buildscript {
+
+    extra["targetSDKVersion"] = 33
+    extra["compileSDKVersion"] = 33
+    extra["minSDKVersion"] = 23
```

Comment:
For now, I am placing versions via extras on the project. We can decide how/where we want to handle the versioning of dependencies across the entire project to ensure version parity

Reply:
do we want to keep min SDK to 23? it could be higher since API 24 or 26 covers more than 80%.

Reply:
Not sure.. I haven't thought that far. We can chat over it in slack and see what the group thinks. I don't feel strongly here. But, If we do want to increase adoption and availability, especially globally, then probably a minSDK of 6 is the way to go

Reply:
minSDK 23 will cause a lot of headache with desugaring when KMP and kotlinx-datetime are involved.

Reply:
Support as low as possible but try to reduce the maintenance only to cover legacy versions.
Always keep in mind, there are regions on earth, where not the latest phone is in everyone’s pocket.

I’m of course fine with raising to 24 or 26 as well. At least when we add video playback to the timeline, we’ll need to raise it most probably.

Reply:
Also, I think using Versions catalog will be easier. 
I can make a PR adding them once this is merged